src/components/tables/hooks.ts

```diff
@@ -1,26 +1,77 @@
-import { useState } from 'react';
+import { useCallback, useMemo } from 'react';
 import { SortDirection } from 'types';
+import {
+    JsonParam,
+    StringParam,
+    useQueryParams,
+    withDefault,
+} from 'use-query-params';
 import { getPagination } from './EntityTable';
 
-function useTableState(defaultSortCol: any, defaultSortDir?: SortDirection) {
+export type TablePrefix =
+    | 'ag' // access grants
+    | 'pr' // prefixes
+    | 'sm' // storage mappings
+    | 'cap' // captures
+    | 'mat' // materializations
+    | 'col' // collections
+    | 'con'; // connectors
```

Comment:
Used so we can have multiple tables on the page at once. The only current use of this is on the access grants page.